code/modules/events/carp_migration.dm

```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/carp_migration
 	name = "Carp Migration"
 	typepath = /datum/round_event/carp_migration
-	weight = 15
+	weight = 10

```

Comment:
if I left carp unchanged and added dolphins with weight 15 then we'd havetwice the chance of carp based events so I split them (approx.)